Classes/Helper/DocumentValidator.php

```diff
@@ -153,7 +153,7 @@ protected function hasAllMandatoryGroupValues(DocumentFormGroup $group, $hasFile
      */
     protected function hasFiles(\EWW\Dpf\Domain\Model\DocumentForm $documentForm)
     {
-        return $documentForm->getPrimaryFile() || $documentForm->getSecondaryFiles();
+        return $documentForm->getFiles();
```

Comment:
Is this function supposed to return boolean? It does not add anything to `$documentForm->getFiles()` so what is the point?